test/unit/range/views/view_minimiser_test.cpp

```diff
@@ -26,11 +28,24 @@ using result_t = std::vector<size_t>;
 using iterator_type = std::ranges::iterator_t< decltype(std::declval<seqan3::dna4_vector&>()
                                                         | seqan3::views::kmer_hash(seqan3::ungapped{4})
                                                         | seqan3::views::minimiser(5))>;
+using two_ranges_iterator_type = std::ranges::iterator_t< decltype(std::declval<seqan3::dna4_vector&>()
+                                                                   | seqan3::views::kmer_hash(seqan3::ungapped{4})
+                                                                   | seqan3::views::minimiser(5,
+                                                                       std::declval<seqan3::dna4_vector&>()
+                                                                       | seqan3::views::complement | std::views::reverse
+                                                                       | seqan3::views::kmer_hash(seqan3::ungapped{4})
+                                                                       | std::views::reverse))>;
```

Comment:
Can you please extract this into its own type alias? That is really hard to read.

Reply:
So I should delete the example? Because I think I added it because it was requested from the reviewer before.
And yes, the minimiser_hash view is going to do the reverse complement stuff internally.